app/views/wiki/edit.html.erb

```diff
@@ -21,6 +21,15 @@
       <li><a href="/wiki/authoring-help#Advanced+formatting"><%= t('wiki.edit.advanced_formatting') %></a></li>
     </ul>
   </div>
+  <hr />
+  <div class="sidebar-nav hidden-sm">
+    <p class="nav-header"><h4><b>Templates</b></h4></p>
+    <p>Use these to start a page from one of these templates:</p>
+    <ul>
+      <li><a href="/post">Method Template lite</a></li>
```

Comment:
At line 29, link would be `<a href="/wiki/new">Method Template</a>` . hi @jywarren, please check if the link is correct. Thanks.

Reply:
It'll be: `/wiki/new?n=14552&title=Your%20method%20name%20here`